releasenotes/notes/paulilist-do-not-broadcast-from-paulis-96de3832fba21b94.yaml

```
@@ -0,0 +1,8 @@
+---
+fixes:
+  - |
+    A bug has been fixed which had allowed broadcasting when a
+    ``PauliList`` is initialized from ``Pauli``s or labels.  For
```

```suggestion
    :class:`.PauliList` is initialized from :class:`.Pauli`\ s or labels.  For
```

(just to get the Sphinx crossrefs, and docutils needs funny things when using letters after backticks)